src/Sulu/Bundle/MediaBundle/Api/Collection.php

```diff
@@ -516,6 +521,36 @@ public function setMediaCount($mediaCount)
 
     /**
      * @VirtualProperty
+     *
+     * @return int The number of sub collections contained by the collection
+     */
+    public function getSubCollectionCount()
+    {
+        return $this->subCollectionCount;
+    }
+
+    /**
+     * @param int $subCollectionCount The new number of sub collections
+     */
+    public function setSubCollectionCount($subCollectionCount)
+    {
+        $this->subCollectionCount = $subCollectionCount;
+    }
+
+    /**
+     * @VirtualProperty
+     *
+     * Returns the total number of all types of sub objects of this collection.
```

Comment:
No, it's of this collection :) If this collection has 2 subfolders and 4 files this method returns 6.